### ISSUE TITLE
Bug - Usage of VueResizable component

### ISSUE THREAD

**Issue by ShiroeSama** [nikitasnv/vue-resizable#35]
# Bug

## Description

I used this package on my VueJs project.
Today when I want to deploy my app, but now I have a white page and nothing seems to load.

I need to deploy some features, some help would be welcomed ^^

## Information

- Project technology : VueJs 2
- Navigator : Mozilla Firefox 98.0.1 (64 bits)
- Package version : 2.1.2 (Latest release)

## Logs / Screeshoots

Here is the error I have via the console.

![image](https://user-images.githubusercontent.com/29405831/158633459-98e668fd-9bcb-4412-9435-6950504e2ae2.png)

**Comment by manoj-rp** [comment]
use this previous version - "vue-resizable": "2.0.5",

**Comment by nikitasnv** [comment]
@ShiroeSama can you reproduce it? Try to import umd version like this `import VueResizable from 'vue-resizable/dist/vue-resizable.umd.min'`

**Comment by ShiroeSama** [comment]
Thanks @manoj-rp, that's the solution I was thinking.
The problem is that my project is made of many small packages and that `vue-resizable` is in several of them, so it's a bit boring.

In the end, I blocked the version at `2.0.5` on all my packages using `vue-resizable`.

@nikitasnv, you'r right I tried to use this import and it's work on `2.1.2`.
But usualy it's been configured like that in the package JSON right ?

Thanks for your help guys !
